README.md

```diff
@@ -53,287 +53,6 @@ and you will find 3 available packages:
-
-### Color
-The UI in this SDK conforms to a colour scheme, which is overridable by creating your own 'KarhooColors' implementation and injecting it to the SDK.
-
-```swift
-struct MyCompanyKarhooColors: KarhooColors { 
-    
-    var primary: UIColor {
-        return .red
-    }
-}
-
-// in your App/SceneDelegate, inject the colors:
-
-KarhooUI.colors = MyCompanyKarhooColors()
-```
-
-
-### Fonts
-Also you can use a custom FontFamily struct to inject your application font to provide a consistent user experience.
-
-```swift
-// full font list: https://gist.github.com/tadija/cb4ec0cbf0a89886d488d1d8b595d0e9
-
-let myAppFontFamily = FontFamily(boldFont: String = "TrebuchetMS-Bold",
-                                 regularFont: String = "TrebuchetMS",
-                                 lightFont: String = "TrebuchetMS-Italic")
-                                 
-KarhooUI.fontFamily = myAppFontFamily
-```
-
-### Injectable routing
-
-Each screen in the UISDK automatically routes to the next. You may want to only use particular screens in the UISDK and custom screens for others. For example you may want to book a trip with the UISDK but use your own address search screen. You can inject a routing implementation to override the navigation flow in the UISDK. To do this you would create your own screen builder and inject it into the SDK.
-
-Create your own version of a screen e.g AddressScreen. (full list of screens are avialable in ```protocol ScreenBuilders```
-
-
-```swift
-// create your own screen
-class MyCustomAddressScreenBuilder: AddressScreenBuilder {
-
-        func buildAddressScreen(locationBias: CLLocation?,
-                                addressType: AddressType,
-                                callback: @escaping ScreenResultCallback<LocationInfo>) -> Screen {
-            return MyCustomAddressViewController()                        
-                                
-        }
-    
-}
-
-struct MyAppRouting: ScreenBuilders {
-
-     var addressBuilder: AddressScreenBuilder {
-        return MyAppAddressScreenBuilder()
-    } 
-}
-
-// AppDelegate / SceneDelegate (setup custom routing)
-KarhooUI.setRouting(routing: MyAppRouting())
-```
-
-Now whenever the user opens the address screen within the UISDK, it will open your address screen implementation when the user comes to enter a pickup or drop off point. 
-
-## Issues
-
-_Looking to contribute?_
-
-### 🐛 Bugs
-
-Please file an issue for bugs, missing documentation, or unexpected behavior.
-
-### 💡 Feature Requests
-
-Please file an issue to suggest new features. Vote on feature requests by adding
-a 👍. This helps maintainers prioritize what to work on.
-
-### ❓ Questions
-
-For questions related to using the library, please re-visit a documentation first. If there are no answer, please create an issue with a label `help needed`.
-
 # Contribution Guide 
```

Comment:
Issue section shouldn't be removed.
Last removed line should be: "Now whenever the user opens the address screen within the UISDK, it will open your address screen implementation when the user comes to enter a pickup or drop off point."